src/DGtal/arithmetic/SternBrocot.ih

```diff
@@ -496,6 +496,139 @@ end() const
   static CFracSequence dummy;
   return ConstIterator( 0, dummy.end() );
 }
+
+template <typename TInteger, typename TQuotient>
+inline
+typename DGtal::SternBrocot<TInteger, TQuotient>::Fraction
+DGtal::SternBrocot<TInteger, TQuotient>::Fraction::
+median(const Fraction & g) const
+{
+  return Fraction(this->p()+g.p(),this->q()+g.q());
+}
+
+
+//----------------------------------------------------------------------------
+template <typename TInteger, typename TQuotient>
+inline
+typename DGtal::SternBrocot<TInteger, TQuotient>::Fraction
+DGtal::SternBrocot<TInteger, TQuotient>::Fraction::
+fractionOfSmallestDenominatorInBetween(const Fraction & other) const
+{
+  Fraction f(*this);  
+  Fraction g(other); 
+  
+  if(f>g)
+    {
+      Fraction tmp = f; f = g; g = tmp;
+    }
+  
+  Fraction res;
+  
+  int i = 0;
+  bool minf;
+  
+  Value uf, ug;
+  ConstIterator itf=f.begin(), itg=g.begin();
+  
+  if(abs(f.p()*g.q()-f.q()*g.p())==1)
```

Comment:
will ```abs``` work if num/denom. are GMP big integer ? (I think there is a DGtal::Abs template functor)